cmd/abidump/main.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019 The go-ethereum Authors
```

Comment:
it's 2020 now

Reply:
but it wasn't when I wrote the code

Reply:
you didn't specify a license when you wrote the code :grin: